Infrastructure/Models/Parcel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Models
+{
+    public class Parcel
```

Comment:
If this just for mock purposes and full Model (more fields) will be implemented in future then OK.

Reply:
Currently it is just for mock purposes.